guide/17-working-with-knowledge-graphs/part3_edit_knowledge_graph.ipynb

```diff
@@ -383,6 +388,56 @@
    "source": [
     "knowledge_graph.graph_property_delete(\"User\", \"name\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Search Indexes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Search indexes can be added to esriFieldTypeString properties to allow the contents of those properties to be search for when using [knowledge_graph.search()](https://developers.arcgis.com/python/api-reference/arcgis.graph.html#arcgis.graph.KnowledgeGraph.search)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# add search indexes to individual properties\n",
+    "knowledge_graph.update_search_index(adds={\n",
+    "    \"User\": {\n",
+    "        \"property_names\": [\"name\"]\n",
+    "    }\n",
+    "})\n",
+    "\n",
+    "# add search indexes to all text properties based on the data model\n",
+    "datamodel = knowledge_graph.datamodel\n",
+    "for entity_type in datamodel['entity_types']:\n",
+    "    prop_list = []\n",
+    "    for prop in datamodel['entity_types'][entity_type]['properties']:\n",
+    "        if datamodel['entity_types'][entity_type]['properties'][prop]['fieldType'] == 'esriFieldTypeString':\n",
+    "            prop_list.append(prop)\n",
+    "    knowledge_graph.update_search_index(adds={entity_type: {\"property_names\": prop_list}})\n",
+    "for entity_type in datamodel['relationship_types']:\n",
+    "    prop_list = []\n",
+    "    for prop in datamodel['relationship_types'][entity_type]['properties']:\n",
+    "        if datamodel['relationship_types'][entity_type]['properties'][prop]['fieldType'] == 'esriFieldTypeString':\n",
+    "            prop_list.append(prop)\n",
+    "    knowledge_graph.update_search_index(adds={entity_type: {\"property_names\": prop_list}})\n",
+    "\n",
+    "# delete a property from the search index\n",
+    "knowledge_graph.update_search_index(deletes={\n",
+    "    \"User\": {\n",
+    "        \"property_names\": [\"name\"]\n",
+    "    }\n",
+    "})"
+   ]
```

Comment:
Reads and looks great locally!
![image](https://github.com/Esri/arcgis-python-api/assets/1399179/58775415-5448-4862-bed8-4feee1b2d642)

Reply:
small typo with search -> searched